profilingdemo/src/main/java/org/kaaproject/kaa/examples/profiling/ProfilingDemoBuilder.java

```diff
@@ -0,0 +1,122 @@
+        CTLSchemaDto endpointProfileCtlSchema = client.saveCTLSchemaWithAppToken(getResourceAsString("endpoint_profile_schema.avsc"),
+                profilingApplication.getTenantId(), profilingApplication.getApplicationToken());
+        CTLSchemaDto configurationCTLSchema = saveCTLSchemaWithAppToken(client, "configuration-schema.avsc",
+                profilingApplication);
+
+        EndpointProfileSchemaDto endpointProfileSchema = new EndpointProfileSchemaDto();
+        endpointProfileSchema.setApplicationId(profilingApplication.getId());
+        endpointProfileSchema.setName("Endpoint profiling endpoint profile schema");
+        endpointProfileSchema.setDescription("Endpoint profile schema describing profiling application profile");
+        endpointProfileSchema.setCtlSchemaId(endpointProfileCtlSchema.getId());
+        endpointProfileSchema = client.saveProfileSchema(endpointProfileSchema);
+        sdkProfileDto.setProfileSchemaVersion(endpointProfileSchema.getVersion());
+
+        // Configuration schema
+        ConfigurationSchemaDto configurationSchema = new ConfigurationSchemaDto();
+        configurationSchema.setApplicationId(profilingApplication.getId());
+        configurationSchema.setName("Endpoint profiling configuration schema");
+        configurationSchema.setDescription("Configuration schema describing profiling application profile");
+        configurationSchema.setCtlSchemaId(configurationCTLSchema.getId());
+        configurationSchema = client.saveConfigurationSchema(configurationSchema);
+
+        logger.info("Configuration schema version: {}", configurationSchema.getVersion());
+        sdkProfileDto.setConfigurationSchemaVersion(configurationSchema.getVersion());
+        logger.info("Configuration schema was created.");
+
+        // Configuration
+        ConfigurationDto baseConfiguration = new ConfigurationDto();
+        baseConfiguration.setApplicationId(profilingApplication.getId());
+        baseConfiguration.setEndpointGroupId(baseEndpointGroup.getId());
+        baseConfiguration.setSchemaId(configurationSchema.getId());
+        baseConfiguration.setSchemaVersion(configurationSchema.getVersion());
+        baseConfiguration.setDescription("Base endpoint profiling configuration");
+
+        String body = FileUtils.readResource(getResourcePath("configuration_devices.json"));
+        logger.info("Configuration body: [{}]", body);
+        baseConfiguration.setBody(body);
+        baseConfiguration.setStatus(UpdateStatus.INACTIVE);
+
+        logger.info("Editing the configuration...");
+        baseConfiguration = client.editConfiguration(baseConfiguration);
+        logger.info("Configuration was successfully edited");
+
+        logger.info("Activating the configuration");
+        client.activateConfiguration(baseConfiguration.getId());
+        logger.info("Configuration was activated");
+
+        logger.info("Finished loading 'Endpoint profiling demo application' data...");
```

Comment:
Add 4 preconfigured endpoint groups according to demo scenario. See epic link to APP-94.